setup.py

```diff
@@ -55,10 +55,9 @@ def get_ec_name_by_system():
         else:
             raise ValueError('Cannot determine architecture')
 
-        return 'ec-{}-{}{}'.format(
+        return 'ec-{}-{}'.format(
```

Comment:
Why did you remove this?

Reply:
For version 2.6.0, the files for windows lack the `.exe` in the filename that was present earlier, so the filenames are now simply consistent with the other systems. See https://github.com/editorconfig-checker/editorconfig-checker/releases/tag/2.6.0 .

Reply:
Cool, thanks for the info.